styles.css

```diff
@@ -0,0 +1,163 @@
+body {
+	font-family: 'Old Standard TT', serif;
+	line-height: 1.4;
+	margin: 0;
+	background: rgba(0, 214, 170, .9);
+	font-weight: 400;
+}
+
+.container {
+	max-width: 960px;
+	margin: auto;
+	overflow: hidden;
+	padding: 0 30px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/burger-one/pullRequest?prid=6050239)